langwatch/src/optimization_studio/hooks/useSocketClient.ts

```
@@ -360,3 +177,174 @@ export const useSocketClient = () => {
+        case "is_alive_response":
+          if (pythonDisconnectedTimeout) {
+            clearTimeout(pythonDisconnectedTimeout);
+            pythonDisconnectedTimeout = null;
+          }
+          debug("Python is alive, setting status to connected");
+          setSocketStatus("connected");
+          break;
+        case "component_state_change":
+          const currentComponentState = getWorkflow().nodes.find(
+            (node) => node.id === message.payload.component_id
+          )?.data.execution_state;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)